ingest/rules/curate.smk

```diff
@@ -116,6 +116,23 @@ rule curate:
         """
 
 
+rule add_genbank_url:
+    input:
+        metadata=temp("data/all_metadata_intermediate.tsv"),
+    output:
+        metadata="data/all_metadata.tsv",
+    log:
+        "logs/add_genbank_url.txt",
+    benchmark:
+        "benchmarks/add_genbank_url.txt",
+    shell:
+        r"""
+        csvtk mutate2 -t \
+          -n url \
+          -e '"https://www.ncbi.nlm.nih.gov/nuccore/" + $accession' \
+          {input.metadata:q} > {output.metadata:q} 2> {log:q}
+        """
```

Comment:
[No changes needed] If the accession column is named "genbank_accession" auspice [will do this for free](https://github.com/nextstrain/auspice/blob/0fea60254e3a5a55692127bafab1b49ca61418d8/src/components/tree/infoPanels/click.js#L90-L94). Your way ("accession" + "url") will also work. The former will have a title of "Genbank accession" while the latter will have "Accession".

Reply:
Thanks for the explanation / confirmation. I went with adding on a `url` field because it felt like less overall churn than re-naming all the places where `accession` was already used.